src/planner/PlanNode.h

```diff
@@ -111,11 +111,20 @@ class PlanNode {
     }
 
     void setOutputVar(std::string var) {
-        outputVar_ = std::move(var);
+        outputVars_.emplace_back(std::move(var));
     }
 
-    std::string varName() const {
-        return outputVar_;
+    std::string outputVar() const {
+        return outputVars_.back();
+    }
+
+    std::string outputVar(size_t index) const {
+        DCHECK_LT(index, outputVars_.size());
+        return outputVars_[index];
+    }
```

Comment:
you can use single `outputVar` function with default parameter `index=0` to keep compatible with old codes.

Reply:
If so there will be a `if`.